app/index.html

```diff
@@ -48,6 +48,7 @@
     <script src="bower_components/ng-file-upload/ng-file-upload.js"></script>
     <script src="bower_components/ng-file-upload/ng-file-upload-shim.js"></script>
     <script src="bower_components/raven-js/dist/raven.js"></script>
+    <script src="bower_components/angular-history/history.js"></script>
```

Comment:
Should we start to slowly move away from bower and use npm instead? Might be nicer on the long run and would completely remove index.html clutter. I was thinking to maybe just `npm install angular-history` and `import 'angular-history';` in `index.js`, or better yet, importing it only locally in your own module.

What do you think?